xtask/src/release.rs

```diff
@@ -1,67 +1,147 @@
-use std::path::Path;
-
+use std::{
+    io::{stdin, stdout, BufRead, Write},
+    path::{Path, PathBuf},
+};
+
+use isahc::{
+    auth::{Authentication, Credentials},
+    config::Configurable,
+    http::StatusCode,
+    HttpClient, ReadResponseExt, Request,
+};
 use itertools::Itertools;
+use semver::Version;
 use serde::Deserialize;
-use serde_json::json;
+use serde_json::{json, Value as JsonValue};
 use toml::from_str as from_toml_str;
-use xshell::{cmd, pushd, read_file};
+use xshell::{pushd, read_file};
 
-use crate::{config, flags, project_root, Result};
+use crate::{cmd, config, Result};
 
-const GITHUB_API_RELEASES: &str = "https://api.github.com/repos/ruma/ruma/releases";
+const CRATESIO_API: &str = "https://crates.io/api/v1/crates";
+const GITHUB_API_RUMA: &str = "https://api.github.com/repos/zecakeh/ruma";
```

Comment:
You committed a reference to your fork again. Are you sure you don't want to make it configurable? 😅

Reply:
I should most of all check my changes one last time before commiting 😅